setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools import find_packages
 from setuptools import setup
 
-version = '1.11.dev0'
+version = '1.2.dev0'
```

Comment:
Please make a 2.0 out of it.
1.2 is smaller than 1.11.

Reply:
done